tasks.c

```diff
@@ -490,7 +493,7 @@ PRIVILEGED_DATA static TaskHandle_t xIdleTaskHandles[ configNUMBER_OF_CORES ];
 /* Improve support for OpenOCD. The kernel tracks Ready tasks via priority lists.
  * For tracking the state of remote threads, OpenOCD uses uxTopUsedPriority
  * to determine the number of priority lists to read back from the remote target. */
-const volatile UBaseType_t uxTopUsedPriority = configMAX_PRIORITIES - 1U;
+static const volatile UBaseType_t uxTopUsedPriority = configMAX_PRIORITIES - 1U;
```

Comment:
This one I am not so sure why we declare this as global instead of static global. Reference this thread
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/188

Reply:
Is it possible to check with @dachalco, the author of #188?
Or any other way to verify this before changing?

Reply:
Looking at the [OpenOCD code](https://sourceforge.net/p/openocd/code/ci/master/tree/src/rtos/FreeRTOS.c) mentioned in the 188 PR, it uses other symbols which are also static, so guess it should be fine.